evennia/accounts/bots.py

```diff
@@ -104,7 +104,7 @@ def basetype_setup(self):
                      "boot:perm(Admin);msg:false();noidletimeout:true()"
         self.locks.add(lockstring)
         # set the basics of being a bot
-        script_key = "%s" % self.key
+        script_key = f"{str(self.key)}"
```

Comment:
Ah. No - there is no need for an f-string here. Just make it `str(self.key)`.

Reply:
Resolved.